src/iocore/cache/StripeSM.cc

```diff
@@ -117,7 +117,7 @@ StripeSM::StripeSM(CacheDisk *disk, off_t blocks, off_t dir_skip, int avg_obj_si
     Stripe{disk, blocks, dir_skip, avg_obj_size, fragment_size},
     fd{disk->fd},
     disk{disk},
-    _preserved_dirs{static_cast<int>(len)}
+    _preserved_dirs{len}
```

Comment:
Using `off_t` seems makes sense because this `Stripe::len` is `off_t`.
https://github.com/apache/trafficserver/blob/41523d163d4285aaccd0dbcdab3bd8aabc2a2a10/src/iocore/cache/Stripe.h#L76

Reply:
I could also look at how big a change it would be to make the table size an `off_t` type, but I need to be careful because I don't understand yet why they were different in the first place. It seems like using the same type for both would be more consistent.

Reply:
This has not been done yet, but I am adding it to my personal TODO.